src/commcare_cloud/manage_commcare_cloud/datadog_monitors.py

```diff
@@ -119,6 +119,27 @@ def get_monitor_definitions(config):
     return monitors
 
 
+def dump_monitor_yaml(monitor_definition):
+    return yaml.safe_dump(monitor_definition, allow_unicode=True)
+
+
+def write_monitor_definition(monitor_id, monitor_definition):
+    if 'env_key' not in monitor_definition:
+        # try to guess it
+        for env_key in ['environment.name', 'host.environment']:
+            if env_key in monitor_definition['message']:
+                monitor_definition['env_key'] = env_key
+                break
```

Comment:
this will just break out of the loop so the next `if` statement will still get executed. Is that intentional?

Reply:
oops yeah you're totally right. Sticking the lines after this in an "else"